site/content/entry/citra-progress-report-2019-q1.md

```diff
@@ -165,149 +165,153 @@ Pinged users will get a notification on their desktop and the message will be hi
 
 {{< youtube -0s65OCmnPg >}}
 
 ### [Status of Scripting Support](https://github.com/citra-emu/citra/pull/4609)
 
-A while back developer [EverOddish](https://github.com/EverOddish) came forward and [implemented](https://github.com/citra-emu/citra/pull/4016) scripting support in Citra using `libZMQ`.
-While it was a very welcome move and was well received by the community, unfortunately `libZMQ` was riddled with building issues.
-The build script for `libZMQ` was huge and incompatible with Citra in many ways.
+A while back, developer [EverOddish](https://github.com/EverOddish) came forward and [implemented](https://github.com/citra-emu/citra/pull/4016) scripting support in Citra using `libZMQ`.
+While it was a very welcome move and was well-received by the community, unfortunately `libZMQ` was riddled with building issues.
+The build script for `libZMQ` was huge, and incompatible with Citra in many ways.
 Also, it seemed to be an unnecessarily big dependency just for scripting.
 
 Unfortunately, [EverOddish](https://github.com/EverOddish) was very busy with other stuff and couldn't work on rewriting it.
-After waiting for a while, [wwylele](https://github.com/wwylele) took over and [reimplemented](https://github.com/citra-emu/citra/pull/4609) scripting over UDP using the [boost asio library](https://think-async.com/Asio/), thus entirely removing `libZMQ` dependency.
-While the current capabilities of scripting is limited to reading and writing memory at this point of time, we plan to expand this in the near future, so stay tuned!
+After waiting for a while, [wwylele](https://github.com/wwylele) took over and [reimplemented](https://github.com/citra-emu/citra/pull/4609) scripting over UDP using the [boost asio library](https://think-async.com/Asio/), thus entirely removing the `libZMQ` dependency.
+While the current capabilities of scripting are limited to reading and writing memory at this point in time, we plan to expand this in the near future, so stay tuned!
 
 ### Status of Official Android App ([#4324](https://github.com/citra-emu/citra/pull/4324), [#4450](https://github.com/citra-emu/citra/pull/4450), [#4575](https://github.com/citra-emu/citra/pull/4575))
 
-When developer SachinVin unofficially ported Citra to Android, our own team members [BreadFish64](https://github.com/BreadFish64) and [jroweboy](https://github.com/jroweboy) were secretly working on an official port.
-Since the app was based on a few months old Citra code, it didn't contain many features, fixes, and performance improvements that were present in latest Citra code.
-This unoffical port, also heavily borrowed UI from Dolphin's android app, which didn't fully meet the requirements of Citra.
+When developer [SachinVin](https://github.com/SachinVin) unofficially ported Citra to Android, our own team members [BreadFish64](https://github.com/BreadFish64) and [jroweboy](https://github.com/jroweboy) were secretly working on an official port.
```

Comment:
Not linking to the unofficial dev's GitHub was an intentional change, as we don't want to redirect users towards the unofficial app.

Reply:
Ah, I did wonder about that. I'll change it back, then.

Reply:
oh geez, i didn't catch this the first time. i wasn't working on android myself. i just did a GLES backend for the dank webgl and citra emscripten memes.